### PR TITLE
Cavium serial settings

### PR DESCRIPTION
**Pull request by NStetskovych**
Add cavium console settings.

Signed-off-by: Nadiya.Stetskovych <Nadiya.Stetskovych@cavium.com>

### CODE REVIEW

File: installer/x86_64/platforms/x86_64-accton_as7512_32x-r0 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ONSOLE_PORT=0x2f8
+CONSOLE_DEV=1
+CONSOLE_SPEED=115200
```
Comment (lguohan) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: can you change this to 9600? all sonic default baud rate is 9600.

Comment (NStetskovych) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: Sorry, but I can`t. Our serial port works on thus rate.

Comment (NStetskovych) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: @lguohan could we leave platform specific speed?

Comment (lguohan) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: It's better not. all sonic should have the same default baud rate speed.

Comment (NStetskovych) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: double checked - it stuck with 9600 speed. I assume it is not correct to connect to console on other speed than it`s work.

Comment (NStetskovych) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: @lguohan how it affects sonic? unfortunately I can`t change it for now.

Comment (NStetskovych) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: will check on which speed we connect through our terminal server.

Comment (NStetskovych) [inline installer/x86_64/platforms/x86_64-accton_as7512_32x-r0]: @lguohan find possibility to change it in BIOS. Tested now it`s fine.

**Pull request merged**